config/foreman.migrations/20210803130619_add_hammer_puppet_plugin.rb

```diff
@@ -0,0 +1 @@
+answers['foreman::cli::puppet'] ||= true
```

Comment:
`||= false` would work well, but `||= true` doesn't. If we do decide to default to `false`, this migration will need modification. I'd prefer to do that ahead of time. From eb73fb7f4b2fe6470816a8707bae433e2b2cd71d:
```ruby
unless answers.key?('foreman::plugin::puppet')
  answers['foreman::plugin::puppet'] = true
end
```

So in this case
```suggestion
unless answers.key?('foreman::cli::puppet')
  answers['foreman::cli::puppet'] = true
end
```